backend/src/main/java/de/kaliburg/morefair/game/round/LadderTypeBuilder.java

```diff
@@ -49,6 +49,13 @@ public LadderTypeBuilder() {
     ladderCostTypeWeights.put(LadderType.CHEAP, 10.f);
     ladderCostTypeWeights.put(LadderType.EXPENSIVE, 10.f);
     ladderCostTypeWeights.put(LadderType.DEFAULT, 100.f);
+
+    ladderGrapesTypeWeights.put(LadderType.BOUNTIFUL, 0f);
+    ladderGrapesTypeWeights.put(LadderType.DROUGHT, 0f);
+    ladderGrapesTypeWeights.put(LadderType.STINGY, 0f);
+    ladderGrapesTypeWeights.put(LadderType.CONSOLATION, 0f);
+    ladderGrapesTypeWeights.put(LadderType.NO_HANDOUTS, 0f);
+    ladderGrapesTypeWeights.put(LadderType.GENEROUS, 0f);
```

Comment:
zero chance of the new ladders appearing outside their respective Round type or Chaos round.

Reply:
do you not need to set default to 100?

Reply:
That's one of the things I was curious what Tobi thought, I landed towards having the main ladder type for each round pretty much become the 'default' ladder type for that round.  ex. On [FARMER] most of the ladders are [BOUNTIFUL] with some of the ladders instead being the negative ladders for the other grape collection methods.

Reply:
oh, I hadn't looked at the lookup table or what it was used for.  That makes more sense.

Reply:
with all weights at 0, won't the inverse lookup table still be emtpy?